core/local/chokidar/initial_scan.js

```diff
@@ -48,6 +48,10 @@ const detectOfflineUnlinkEvents = async (
   let unappliedMoves = []
 
   for (const doc of docs) {
+    if (metadata.needsRevMigration(doc)) {
+      await metadata.migrateRev(doc, { pouch })
```

Comment:
Maybe we could `pouch` directly?
(I suppose there's no reason we would need anything else in `.migrateRev()`)

Reply:
It was convenient to pass the options this way, especially in Atom's dispatch step.
But I'm fine with passing `Pouch` only.